xbmc/platform/linux/AppParamParserWebOS.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+ *  Copyright (C) 2005-2023 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "AppParamParserWebOS.h"
+
+CAppParamParserWebOS::CAppParamParserWebOS()
+{
+}
+
+CAppParamParserWebOS::~CAppParamParserWebOS() = default;
+
+void CAppParamParserWebOS::ParseArg(const std::string& arg)
+{
+  // On webOS the app launcher adds a JSON as the first argument by default. These arguments do not contain
+  // useful information. So to avoid kodi trying to parse these args and misinterpret them as a video file.
+  if (m_nArgs++ == 0 && arg.front() == '{' && arg.back() == '}')
```

Comment:
You should check the length of `arg` before accessing it. => `arg.size() > 1`. 
`front()`and `back()`will most probably crash if called on an empty string.

Reply:
Done!